rholang/src/main/scala/coop/rchain/rholang/interpreter/compiler/normalizer/processes/PMatchNormalizer.scala

```diff
@@ -25,10 +24,10 @@ object PMatchNormalizer {
     }
 
     for {
-      targetResult <- normalizeMatch[F](p.proc_, input.copy(par = VectorPar()))
+      targetResult <- normalizeMatch[F](p.proc_, input.copy(par = NilN))
       cases        <- p.listcase_.asScala.toList.traverse(liftCase)
 
-      initAcc = (Vector[MatchCase](), targetResult.freeMap, BitSet(), false)
+      initAcc = (Seq[MatchCaseN](), targetResult.freeMap)
```

Comment:
Default implementation for Seq is List which takes [linear time](https://docs.scala-lang.org/overviews/collections-2.13/performance-characteristics.html) for append. Vector takes constant time for both prepend and append. So be careful when changing Vector to Seq. If append is used in this collection, it will be slower.

Reply:
We should use underlying Vector as instance for Seq everywhere like in the old proto types.